Cartfile

```diff
@@ -0,0 +1,2 @@
+github "tonymillion/Reachability" "master"
```

Comment:
There's a warning about this as part of its repo: https://github.com/tonymillion/Reachability#warning-there-have-been-reports-of-apps-being-rejected-when-reachability-is-used-in-a-framework-the-only-solution-to-this-so-far-is-to-rename-the-class

Is this necessary as part of the Cartfile or is it something that could optionally be added?